core/src/com/unciv/logic/civilization/PolicyManager.kt

```diff
@@ -181,11 +189,8 @@ class PolicyManager {
         }
 
         for (unique in policy.uniques) {
-            if (unique == "Triggers a global alert") {
```

Comment:
Why did you delete this?

Reply:
you can search this word in policy, it have not been used.🌝🌝

Reply:
I added it as an alternative to the other unique below, when you don't want some extra text to show up, so I would keep it for moddability

Reply:
@xlenstra These can be replaced with "Triggers the following global alert: [param]" if we give a special "[param]"
by the way, don't mix the mod needed code in the original branch😄